Adding my keyboard layout (#13939)
qmk/qmk_firmware: keyboards/cest73/tkm/readme.md

# Ten Key Mirrored - 2021

> ![TKM-2021](https://geekhack.org/index.php?action=dlattach;topic=110499.0;attach=272800;image)

Reviewer: Could you please reupload this to Imgur?

> ![TKM-2021](https://imgur.com/AlvRk1z.jpg)
> 
> 
> Another take on left side numpad and this time mirrored too for the ever more spacious mouse playground.
> 

Reviewer: ```suggestion
![TKM-2021](https://imgur.com/AlvRk1zh.jpg)

Another take on left side numpad and this time mirrored too for the ever more spacious mouse playground.
```

* Keyboard Maintainer: cest73 (aka fire-h0und)
* Hardware Supported: TKM-2021 (multiple layouts and switch options including ALPS, ISO and BAE)
* Hardware Availability: https://geekhack.org/index.php?topic=110499

> 
> Make example for this keyboard (after setting up your build environment):
> 
>     qmk compile -kb cest73/tkm -km default

Reviewer: ```suggestion
Make example for this keyboard (after setting up your build environment):

    make cest73/tkm:default
```

# Ten Key Mirrored - 2021

![TKM-2021](https://geekhack.org/index.php?action=dlattach;topic=110499.0;attach=272800;image)

Another take on left side numpad and this time mirrored too for the ever more spacious mouse playground.

> 
> Keyboard Maintainer: cest73 (aka fire-h0und)
> 
> Hardware Supported: TKM-2021 (multiple layouts and switch options including ALPS, ISO and BAE)
> 
> Hardware Availability: https://geekhack.org/index.php?topic=110499
> 

Reviewer 2: ```suggestion

* Keyboard Maintainer: cest73 (aka fire-h0und)
* Hardware Supported: TKM-2021 (multiple layouts and switch options including ALPS, ISO and BAE)
* Hardware Availability: https://geekhack.org/index.php?topic=110499
```

Make example for this keyboard (after setting up your build environment):

    qmk compile -kb cest73/tkm -km default

> As the board uses Teensy 2/0 as a daughter board, to bring the board into flash mode is it sufficient to press the botton present on the Teensy.

Reviewer 3: ```suggestion
As the board uses Teensy 2.0 as a daughter board, to bring the board into flash mode is it sufficient to press the button present on the Teensy.
```

[TKM-2021 PCB](https://geekhack.org/index.php?action=dlattach;topic=110499.0;attach=268641;image)

> Despite many hours gazing over the schematis there There was a nasty error on the first batch: namely the 6 and 7 columns got swapped in the design process. Luckily it was trivial to fix in the frimware section (see the comments in the layout file(s)).

Reviewer 3: ```suggestion
Despite many hours gazing over the schematics there was a nasty error on the first batch: namely the 6 and 7 columns got swapped in the design process. Luckily it was trivial to fix in the firmware section (see the comments in the layout file(s)).
```
